ChangeLog

```diff
@@ -10,6 +10,7 @@
    * Change: Factories being destroyed could cause annoying asserts. Now set to quiet warnings (commit:616f142c6a031fc36aceb05f8c40156c563225bb)
    * Change: Improved Windows installers (ticket:4748)
    * Change: Location of configuration directory (commit:54901bcd1672c9e22db35c66741ec2eda123b491)
+   * Change: uint8_t to uint32_t variables in serializer for support some mods (commit:c05e4f7851d7f6ab7631b11b81ee9c333cab58cd, commit:3a729d9568b1a06e733651efc08003e601b5846c, ticket:4677)
```

Comment:
The Changelog is a document for ordinary people. This kind of comment is completely unintelligible to non-programmers, and does not belong there.

Reply:
Removed that line. A more simple description existed farther in, missing the commits, so I added them to it.

Reply:
Ok thanks, got it.